config/settings.yml

```diff
@@ -1480,6 +1480,6 @@
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
     :websocket_worker:
-       :connection_pool_size: 8
+       :connection_pool_size: 26
```

Comment:
Hm...there is one more possible corner-case, where we might need one more connection. The internal thread of the websocket<->socket proxy will eat a DB connection after a remote console [gets disconnected](https://github.com/ManageIQ/manageiq/blob/master/lib/websocket_proxy.rb#L54) (i.e. we need to delete the `SystemConsole` record on disconnect). Then if a new connection arrives, the Rack middleware [will find](https://github.com/ManageIQ/manageiq/blob/master/lib/websocket_server.rb#L66) an empty connection pool. I don't think that we can try this out, but theoretically it's possible...